agent/agent.go

```diff
@@ -673,7 +673,7 @@ func (a *Agent) runPreAndMain(ctx context.Context, tc *taskContext) (status stri
 		a.jasper,
 		defaultStatsInterval,
 		"uptime",
-		"df -h",
+		"df -h -i",
```

Comment:
So actually, it looks like running `df -h -i` on Linux machines will suppress the block size stats and only print the inode stats, whereas, for Unix-based systems, `df -h` already prints both the block size and inode stats (and does the same when you add the `-i` flag). 

Printing `df -h -i` on a separate line would be the easiest solution, but on the Unix-based systems it would print the same thing twice adding a significant amount of noise to these logs. We _could_ check the OS and only add the `df -h -i` command if it is linux (and maybe windows? not sure what happens with cygwin).

Reply:
@julianedwards perhaps 655b27bca will solve the problem? I'm open to suggestions!